Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.9-alpine as builder
+
+RUN apk add -U ca-certificates
```

Comment:
I feel like this should go into the actual image.

Reply:
yep good call